operator/v1/types_klusterlet.go

```diff
@@ -278,6 +286,42 @@ type WorkAgentConfiguration struct {
 	AppliedManifestWorkEvictionGracePeriod *metav1.Duration `json:"appliedManifestWorkEvictionGracePeriod,omitempty"`
 }
 
+// Leader election configuration
+type LeaderElectionConfig struct {
+	// Disable leader election.
```

Comment:
do we really need to expose every configurations here?

Reply:
Since the leader election will always be enabled in OCM single node case, and we are not 100% sure if it will impact the user's cases, it might be useful to provide a disable option in `Klusterlet`.